examples/dataproc/main.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import sys
+import time
+
+from google.protobuf.field_mask_pb2 import FieldMask
+
+from yandex.cloud.vpc.v1.network_service_pb2 import ListNetworksRequest
+from yandex.cloud.vpc.v1.network_service_pb2_grpc import NetworkServiceStub
+from yandex.cloud.vpc.v1.subnet_service_pb2 import ListSubnetsRequest
+from yandex.cloud.vpc.v1.subnet_service_pb2_grpc import SubnetServiceStub
+from yandex.cloud.iam.v1.service_account_service_pb2 import ListServiceAccountsRequest
+from yandex.cloud.iam.v1.service_account_service_pb2_grpc import ServiceAccountServiceStub
+
+
+import yandex.cloud.dataproc.v1.common_pb2 as common_pb
```

Comment:
Shouldn't we merge all `yandex.cloud.dataproc.v1.*` import blocks?

Reply:
I see no reason to do it. But can fix if you insist.

Reply:
I just wanted to know the logic behind this division.

Reply:
Next time I'll add another block with JobService, so there will be more of them.
Dataproc is the most complicated service among mdb-related, and because main purpose of this file is to be as readable ass possible I decided that It would be ok to note that there are several different subservices